integration/test/DutchOrder.spec.ts

```diff
@@ -123,10 +123,53 @@ describe('DutchOrder', () => {
       .true;
     expect(builtOutput.recipient).to.eq(swapperAddress);
 
-    order = preBuildOrder.nonFeeRecipient(ethers.constants.AddressZero).build();
+    order = preBuildOrder.nonFeeRecipient(ethers.constants.AddressZero, ethers.constants.AddressZero).build();
```

Comment:
wait lol there should be a condition that u cant pass in the fee recipient address as the new non-fee recipient address

Reply:
ok fair